jobs/JRRFS_ENSMEAN

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LINENO}]: '
+set -x
+date
+#
+#-----------------------------------------------------------------------
+# Specify Execution Areas
+#-----------------------------------------------------------------------
+#
+export HOMErrfs=${HOMErrfs} #comes from the workflow at runtime
+export EXECrrfs=${EXECrrfs:-${HOMErrfs}/exec}
+export FIXrrfs=${FIXrrfs:-${HOMErrfs}/fix}
+export PARMrrfs=${PARMrrfs:-${HOMErrfs}/parm}
+export USHrrfs=${USHrrfs:-${HOMErrfs}/ush}
+#
+#-----------------------------------------------------------------------
+#  find group and model forecast information
+#-----------------------------------------------------------------------
+#
+export GROUP_INDEX=${GROUP_INDEX:-1}
+export GROUP_TOTAL_NUM=${GROUP_TOTAL_NUM:-1}
+export HISTORY_INTERVAL=${HISTORY_INTERVAL:-1}
+export ENS_SIZE=${ENS_SIZE:-1}
+#
+#-----------------------------------------------------------------------
+# Obtain unique process id (pid) and create the run directory (DATA).
+#-----------------------------------------------------------------------
+#
+export pid=${pid:-$$}
+export MEMDIR="/ensmean"
+export UMBRELLA_SAVE_FCST_DATA=${UMBRELLA_SAVE_FCST_DATA:-${DATAROOT}/${RUN}_save_fcst_${cyc}_${rrfs_ver}/${WGF}}
+export jobid=${jobid:-ensmean_g${GROUP_INDEX}_${cyc}}
+export UMBRELLA_ENSMEAN_DATA=${UMBRELLA_ENSMEAN_DATA:-${DATAROOT}/${RUN}_save_fcst_${cyc}_${rrfs_ver}/${WGF}${MEMDIR}}
+export DATA=${DATA:-${UMBRELLA_ENSMEAN_DATA}/${jobid}}
+
+if [[ -d "${DATA}" ]]; then # remove the ${DATA}/${GROUP_INDEX} directory if existed
```

Comment:
The comment is not accurate. 
We remove the `${DATA}` directory, not `${DATA}/${GROUP_INDEX}`
`${DATA}` is the job run directory, it includes `${GROUP_INDEX}`